source/fuzz/transformation_add_no_contraction_decoration.cpp

```diff
@@ -37,6 +37,11 @@ bool TransformationAddNoContractionDecoration::IsApplicable(
   if (!instr) {
     return false;
   }
+  opt::BasicBlock* cur_block = ir_context->get_instr_block(instr);
+  // The instruction must have a block.
+  if (cur_block == nullptr) {
+    return false;
+  }
```

Comment:
I don't think this is actually needed, as the IsArithmetic should restrict to the valid cases. My previous comment was mainly about checking that there was not another transformation getting the block of an instruction and then using it without checking for nullptr.

Reply:
I agree - please revert this, @antnkarp.

Reply:
Done

Reply:
Done

Reply:
@antnkarp Thanks - please remember to resolve a comment once you've addressed it - only leave it open if you are unsure how to address it or if you have posted a follow-up question.  So you can resolve this one, and please go up and resolve any above comments that you have addressed.